test-cases/features/repair-based-operations/longevity-100gb-4h-ReplaceNode.yaml

```diff
@@ -20,7 +20,7 @@ nemesis_class_name: 'NodeTerminateAndReplace'
 nemesis_interval: 1
 nemesis_during_prepare: false
 nemesis_filter_seeds: false
-seeds_num: 1
```

Comment:
Can we disable nemesis on seed by setting `nemesis_filter_seeds` to True ?

Reply:
Yes, we can, but as i understood, goal to test with and without seeds

Reply:
This is a good idea. I suggested ignoring the seed nodes completely in the thread with Alex where he had a fix for the seed nodes in sct too. I did not know there is a nemesis_filter_seeds for it though.